.gitignore

```diff
@@ -26,4 +26,6 @@ local.properties
 .loadpath
 
 # Mac OS files
-.DS_Store
\ No newline at end of file
+.DS_Store
+omod/src/main/java/org/openmrs/module/patientportaltoolkit/fragment/controller/EditCancerCommunityResourcesFragmentController.java
```

Comment:
These individual files should not be included in gitignore. If you do not want to check in changes uncheck the checkbox while committing